upload/system/library/cart.php

```diff
@@ -443,4 +443,16 @@ public function hasDownload() {
 
 		return $download;
 	}
+
+	public function hasCart() {
```

Comment:
*isEmpty()* would be more logical name, comparing $this->cart->hasCart() and $this->cart->isEmpty()